core/src/main/java/com/linecorp/armeria/common/metric/MeterIdPrefixFunction.java

```diff
@@ -71,26 +71,12 @@ static MeterIdPrefixFunction ofDefault(String name) {
         return new MeterIdPrefixFunction() {
             @Override
             public MeterIdPrefix activeRequestPrefix(MeterRegistry registry, RequestLog log) {
-                return new MeterIdPrefix(name, buildTags(log));
-            }
-
-            @Override
-            public MeterIdPrefix apply(MeterRegistry registry, RequestLog log) {
-                final List<Tag> tags = buildTags(log);
-
-                // Add the 'httpStatus' tag.
-                final HttpStatus status;
-                if (log.isAvailable(RequestLogAvailability.RESPONSE_HEADERS)) {
-                    status = log.status();
-                } else {
-                    status = HttpStatus.UNKNOWN;
-                }
-                tags.add(Tag.of("httpStatus", status.codeAsText()));
-
-                return new MeterIdPrefix(name, tags);
+                final ImmutableList.Builder<Tag> tagListBuilder = ImmutableList.builderWithExpectedSize(4);
+                buildTags(tagListBuilder, log);
+                return new MeterIdPrefix(name, tagListBuilder.build());
             }
 
-            private List<Tag> buildTags(RequestLog log) {
+            private void buildTags(ImmutableList.Builder<Tag> tagListBuilder, RequestLog log) {
```

Comment:
How about moving `buildTags()` after `apply()` because it's used by both `activeRequestPrefix()` and `apply()`?